### PR TITLE
RR-954: Team test changes

### PR DESCRIPTION
**Pull request by lynnette-orr**
- Moved the goals columns to the left
- Changed In progress status tag from green to blue
- For FS initial assessments -  “assessed on… “ aligned to the right and the level next to the subject
- For courses and qualifications - remove column names
- Vertically align 'Add a goal' and target icon in actions card
- Print button activated for pre-induction

Screenshot 2024-10-10 at 16 43 39

### CODE REVIEW

File: server/routes/overview/mappers/functionalSkillsMapper.test.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ describe('functionalSkillsMapper', () => {
       {
         prn: prisonNumber,
         establishmentId: 'MDI',
-        establishmentName: 'MOORLAND (HMP & YOI)',
+        establishmentName: 'Moorland (HMP & YOI)',
```
Comment (nathanrussell-moj-digital) [inline server/routes/overview/mappers/functionalSkillsMapper.test.ts:14]: No, this is not right. These values are ALWAYS uppercase in the data we get back from Curious (which is why we do a prison name lookup)

File: server/routes/overview/mappers/functionalSkillsMapper.ts [hunk 2]
```diff
@@ -28,6 +28,12 @@ const dateOrNull = (value: string): Date | undefined => {
   return value ? startOfDay(parseISO(value)) : undefined
 }
 
+const formatPrisonName = (prisonName: string) => {
```
Comment (nathanrussell-moj-digital) [inline server/routes/overview/mappers/functionalSkillsMapper.ts:31]: No, why is the mapper formatting the prison name? We dont want to do this here (formatting the prison name is not always as easy as lower casing / upper casing as you have here). This is why we do a lookup from the prison-register API

File: server/services/curiousService.test.ts [hunk 3]
```diff
@@ -148,7 +148,7 @@ describe('curiousService', () => {
             assessmentDate: moment('2012-02-16').utc().toDate(),
             grade: 'Level 1',
             prisonId: 'MDI',
-            prisonName: 'MOORLAND (HMP & YOI)',
+            prisonName: 'Moorland (HMP & YOI)',
```
Comment (nathanrussell-moj-digital) [inline server/services/curiousService.test.ts:151]: Why this change? 🤔

Comment (lynnette-orr) [reply server/services/curiousService.test.ts:154]: Reformatted to use the lookup.

Comment (nathanrussell-moj-digital) [reply server/services/curiousService.test.ts:154]: I'm not sure I understand why though? 🤔 
Lets have a quick chat when you get online 👍

**Review by nathanrussell-moj-digital — APPROVED** [review]
LGTM 👍
